src/runtime/config/configuration-clh.toml.in

```diff
@@ -15,6 +15,33 @@ path = "@CLHPATH@"
 kernel = "@KERNELPATH_CLH@"
 image = "@IMAGEPATH@"
 
+# Enable confidential guest support.
+# Toggling that setting may trigger different hardware features, ranging
+# from memory encryption to both memory and CPU-state encryption and integrity.
+# The Kata Containers runtime dynamically detects the available feature set and
+# aims at enabling the largest possible one.
+#
+# Known limitations:
+# * Does not work by design:
+#   - CPU Hotplug 
```

Comment:
My own curiosity -- is this a policy decision, or is there a limitation in the VMM / guest that prohibits hot plug?

Reply:
My understanding is that it's not supported by the TEE.  It's not a limitation in the VMM / guest, but rather something not supported by Confidential Computing in general.